EMULATOR/EMULATOR.C

```diff
@@ -154,8 +159,8 @@ void Emulator_Init(char* windowName, int screen_width, int screen_height)
 		fprintf(stderr, "Error: %s\n", glewGetErrorString(err));
 	}
 
-	SDL_memset(&vram, 0, sizeof(1024 * 512 * 2));
-	SDL_GL_SetSwapInterval(-1);
+	SDL_memset(&vram, 0, sizeof(VRAM_WIDTH * VRAM_HEIGHT * sizeof(unsigned short)));
```

Comment:
Does this mean no matter what happens the game will run in a constant output resolution?

Reply:
Yes, for now.